test/common/toString.spec.ts

```diff
@@ -47,3 +47,29 @@ describe('global function patch', () => {
        }));
   });
 });
+
+describe('ZoneTask', () => {
+  it('should return handleId.toString if handleId is available', () => {
+    let macroTask: any = undefined;
+    let microTask: any = undefined;
+    const zone = Zone.current.fork({
+      name: 'timer',
+      onScheduleTask: (delegate: ZoneDelegate, curr: Zone, target: Zone, task: Task) => {
+        if (task.type === 'macroTask') {
+          macroTask = task;
+        } else if (task.type === 'microTask') {
+          microTask = task;
+        }
+        return task;
+      }
+    });
+    zone.run(() => {
+      const id = setTimeout(() => {});
+      clearTimeout(id);
+      Promise.resolve().then(() => {});
+      expect(typeof macroTask.toString()).toEqual('string');
+      expect(macroTask.toString()).toEqual(id.toString());
+      expect(typeof microTask.toString()).toEqual('string');
+    });
```

Comment:
I think it would be valueable to also verify that two different macrotasks have different string representations (just in case).

Reply:
Got it, case added.